rust/driver_manager/src/lib.rs

```diff
@@ -381,6 +384,22 @@ impl ManagedDriver {
         let default_entrypoint = get_default_entrypoint(filename.as_ref());
 
         let entrypoint = entrypoint.unwrap_or(default_entrypoint.as_bytes());
+        // By default, go builds the libraries with '-Wl -z nodelete' which does not
+        // unload the go runtime. This isn't respected on mac ( https://github.com/golang/go/issues/11100#issuecomment-932638093 )
+        // so we need to explicitly load the library with RTLD_NODELETE( which prevents unloading )
+        #[cfg(target_os = "macos")]
+        let library: libloading::Library = unsafe {
+            const RTLD_LAZY: i32 = 0x1;
+            const RTLD_LOCAL: i32 = 0x4;
+            const RTLD_NODELETE: i32 = 0x80;
```

Comment:
I guess we could pull these from the `libc` crate, is it worth it though? cc @eitsupi

Reply:
It seems that `RTLD_LAZY` and `RTLD_LOCAL` are also defined in libloading? 🤔
https://github.com/nagisa/rust_libloading/blob/dab97c569b33bd515e16637b8dedbdc696d9ec9c/src/os/unix/consts.rs

Reply:
Yeah. Only those two, but not `RTLD_NODELETE`. I don't feel super stressed about hard-coding them here to avoid taking on libc as a dep.